src/styles/calendar.scss

```diff
@@ -0,0 +1,58 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
+  box-sizing: border-box;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition:
+      background-color 500ms,
+      transform 500ms;
+
+    &::before {
+      font-family: Arial, sans-serif;
+      font-size: 30px;
+      content: '';
```

Comment:
The `&::before` pseudo-element has an empty `content` property which is then overridden in a loop. It's better to remove this default empty `content` property since it's not serving any purpose.